al2/x86_64/standard/3.0/Dockerfile

```
@@ -398,7 +398,7 @@ RUN set -ex \
     && pip3 install --no-cache-dir --upgrade setuptools wheel aws-sam-cli awscli boto3 pipenv virtualenv --use-feature=2020-resolver
 
 # Node 12
-ENV NODE_12_VERSION="12.19.1"
+ENV NODE_12_VERSION="12.21.0"
```

```suggestion
ENV NODE_12_VERSION="12.22.2"
```

Blog post on July 2021 Security Releases: https://nodejs.org/en/blog/vulnerability/july-2021-security-releases/